src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibleBuffer.ts

```diff
@@ -87,6 +88,12 @@ export class AccessibleBufferWidget extends DisposableStore {
 				this._font = _xterm.getFont();
 			}
 		}));
+		this.add(this._xterm.raw.onData(() => {
```

Comment:
`raw.onData` will only fire when the user inputs into the terminal

Reply:
oh nvm I'm wrong I'm refocusing the buffer to see the changes